### PR TITLE
Basic a11y

### PR DESCRIPTION
**Pull request by liamross**
First pass at making buttons show tab indicators, and incorporating the toolkit in for checkboxes (right now only in search overlay).

### CODE REVIEW

**Comment by liamross** [comment]
We should probably squash this when we merge it, the only reason so many commits are showing up is cause @kristianhein 's new ui branch was probably squashed into master, and this is technically branched off of that branch prior to it merging with master

File: src/components/Input/Input.js [hunk 1]
```diff
@@ -1,12 +1,11 @@
 import React from 'react';
-import { useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 import Icon from 'components/Icon';
+import { useSelector } from 'react-redux';
```
Comment (ZhijieZhang) [inline src/components/Input/Input.js:4]: You don't like where it was? Just curious, no need to revert the change.

File: src/components/SearchOverlay/SearchOverlay.js [hunk 2]
```diff
@@ -459,24 +462,6 @@ class SearchOverlay extends React.PureComponent {
...
-    this.executeDebouncedFullSearch();
-  }
-
-  onChangeWildcard = e => {
-    this.props.setWildcard(e.target.checked);
-    this.clearSearchResults();
-    this.executeDebouncedFullSearch();
-  }
-
-  onChangeWildcard = e => {
```
Comment (ZhijieZhang) [inline src/components/SearchOverlay/SearchOverlay.js:474]: Good catch!

File: src/components/SearchOverlay/SearchOverlay.js [hunk 3]
```diff
@@ -5,8 +5,6 @@ import { connect } from 'react-redux';
 import onClickOutside from 'react-onclickoutside';
 
 import Icon from 'components/Icon';
-import Tooltip from 'components/Tooltip';
-import Input from 'components/Input';
```
Comment (ZhijieZhang) [inline src/components/SearchOverlay/SearchOverlay.js:9]: Do we want to use the toolkit for the `PrintModal` as well? That component uses `Input` as well.

**Comment by ZhijieZhang** [comment]
Nothing is stopping this PR from being merged. Just have few questions.

File: src/components/SearchOverlay/SearchOverlay.js (hunk 3 above)
Comment (liamross) [reply src/components/SearchOverlay/SearchOverlay.js:9]: Yeah I'll do that one next, just slightly more complexity with it lining up properly

File: src/components/Input/Input.js (hunk 1 above)
Comment (liamross) [reply src/components/Input/Input.js:4]: haha this one was an accident, I sometimes instinctively hit <kbd>Shift</kbd> + <kbd>⌘</kbd> + <kbd>O</kbd> which organizes the imports, let me revert

**Comment by ZhijieZhang** [comment]
LGTM, thanks!
